web/components/InputField.module.scss

```diff
@@ -46,10 +28,15 @@
         padding: 8px 12px;
         color:  var(--light-color);
     }
-}
 
-.buttonSet {
-    display: flex;
-    flex-direction: row;
-    align-items: flex-start;
-}
\ No newline at end of file
+    select {
+        border: 1px solid #E3E8F0;
+        box-sizing: border-box;
+        border-radius: 8px;
+        width: 100%;
+        font-size: 14px;
+        background: #FFF;
```

Comment:
You allow yourself to set colours outside of variables?

Reply:
This was definitely a bit lazy..fixing.